h2o-automl/src/main/java/ai/h2o/automl/events/EventLogEntry.java

```diff
@@ -62,17 +63,17 @@ public Object parseObject(String source, ParsePosition pos) {
     }
   }
 
-  static final Format epochFormat = new SimpleFormat<Date>() {
+  public static final Format epochFormat = new SimpleFormat<Date>() {
```

Comment:
is this okay to have it static?

I am refering to Format's javadoc:
``
 * Formats are generally not synchronized.
 * It is recommended to create separate format instances for each thread.
 * If multiple threads access a format concurrently, it must be synchronized
 * externally.
``

Reply:
@michalkurka Mmm, fair point! Never realized that about Format...
Will create a ticket to fix this: we haven't had any issue with this until now.

I think this could easily be fixed by just putting those in a ThreadLocal.

Reply:
https://0xdata.atlassian.net/browse/PUBDEV-7126